sematic/ee/__init__.py

```diff
@@ -1,12 +0,0 @@
-# Sematic
```

Comment:
Other projects with "extras" seem to nest the stuff that requires the extra under an appropriately named module. So I moved the Ray stuff to be importable under `sematic.ee.ray`